client/main.lua

```diff
@@ -22,9 +21,10 @@ local function drawPlayerNumbers()
                 local player = players[i]
                 local serverId = GetPlayerServerId(player.id)
                 if shouldShowPlayerId(playerOptin[serverId].isOnDutyAdmin) then
+                    local pedCoords = GetEntityCoords(player.ped)
```

Comment:
fixes this
![obraz](https://github.com/Qbox-project/qbx_scoreboard/assets/36608116/0b035aad-3023-4af9-8801-a30ada291bff)